app/Repositories/User/Criterias/GetTastesByUserWithCriteria.php

```diff
@@ -16,6 +16,12 @@ public function __construct(int $userId)
 
     public function apply($model, RepositoryInterface $repository)
     {
-        return $model->where('user_id', $this->userId)->orWhere('is_default', true);
+        $userId = $this->userId;
```

Comment:
remove, $this->user_id will be visible inisde callback

Reply:
$this->user_id unreachable statement error